### PR TITLE
fixes #514

### LINKED ISSUE #514: Relabeling Actions and Projects during agenda processing
## Issue - Enhancement

2 of 2 users tests demonstrated confusion around the difference between projects and actions. After the test, I asked them to tell me what they thought a "Private Action" and a "Team Project" would do.

Folks told me a "Team Project" probably goes on the team's board. A "Private Action" they thought sounded like it would be visible only to them, but, they weren't sure in the current design where they would go to see such a thing. I imagine that could be cleared up a bit by #274.

What do you think about:

- New Action (shortcut: `A`) -> New Private Action (shortcut: `P`)
- New Project (shortcut `P`) -> New Team Project (shortcut: `T`)

- **Estimated effort:** 1 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

### PR DESCRIPTION
**Pull request by ackernaut**
[new CreateCards in action](http://g.recordit.co/EvmErj0n3V.gif)

Also, fixes the maxWidth of the CreateCard to be uniform with UI const.

### CODE REVIEW

File: src/universal/components/CreateCard/CreateCard.js [hunk 1]
```diff
@@ -80,18 +69,41 @@ CreateCard.propTypes = {
   styles: PropTypes.object
 };
 
-const labelStyles = {
-  display: 'inline-block',
+const borderRadius = '.1875rem';
```
Comment (mattkrick) [inline src/universal/components/CreateCard/CreateCard.js]: genuinely curious, why is border radius in rem instead of pixels or %?

Comment (ackernaut) [inline src/universal/components/CreateCard/CreateCard.js]: A lot of our defaults are based on `1rem` which is `16px` —using certain sizes can keep things consistent (and are supposed to scale up/down via zoom nicely)...

Comment (ackernaut) [inline src/universal/components/CreateCard/CreateCard.js]: That is `3px`... :)

Comment (mattkrick) [inline src/universal/components/CreateCard/CreateCard.js]: but if someone increases their default browser font size, then this means the corners will look more rounded, right?

Comment (ackernaut) [inline src/universal/components/CreateCard/CreateCard.js]: Yes, but in theory, everything should scale up (zoom style). And I believe we are setting the default `16px` on the body or html.

Comment (ackernaut) [inline src/universal/components/CreateCard/CreateCard.js]: I think most modern browser just default to zoom now, so it really doesn’t matter—they will scale up from `px` or `rem`...

Comment (mattkrick) [inline src/universal/components/CreateCard/CreateCard.js]: alrighty, i'm cool with whatever you decide, just for fun though, here's what happens in chrome when you set font to extra large: http://recordit.co/pTBinNyjey

Comment (ackernaut) [inline src/universal/components/CreateCard/CreateCard.js]: I know what you mean. Here is what I had in mind:

http://recordit.co/y4onoyGUey

I’ve (mostly) used the `rem` unit, so changing the `16px` on the `html` selector is mostly like a zoom. Margins, paddings, border-radius, font-size all change based on that base multiplier.

What we could do, and would be interesting when we start exploring adaptive/responsive/mo-mobile-first, is use something fixed for “UI chrome” elements (certain panels, borders, etc.) and use `rem` for text and elements that need the proper sizing/spacing. Basically the chrome wouldn’t change but the other elements would get larger/smaller. This is all based on the idea of letting the user set their default browser font-size. We would remove the font-size from html/body.

**Pull request merged**